daml-foundations/daml-ghc/ghc-compiler/src/DA/Daml/GHC/Compiler/Config.hs

```diff
@@ -76,6 +76,7 @@ wOptsSet =
   , Opt_WarnPrepositiveQualifiedModule
   , Opt_WarnOverlappingPatterns
   , Opt_WarnIncompletePatterns
+  , Opt_WarnMissingMethods
```

Comment:
I wonder if we should make this a fatal warning.

Reply:
I wondered that too... I'll change. It feels like missing fields, which is an error.

Reply:
Turned it into an error.